Build Faild when disable LV_LINUX_FBDEV_MMAP

### LVGL version

v9.3.0-883-g0648b6d91

### Platform

FBDEV Linux x86-64

### What happened?

 I set "LV_LINUX_FBDEV_MMAP 0" in build config file then run build script ,I got a build failed message
```BASH
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_14.c.o
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_14_aligned.c.o
In file included from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_assert.h:17,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_color.h:17,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/lv_display.h:19,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.h:17,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:9:
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c: In function ‘write_to_fb’:
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:275:42: error: ‘errno’ undeclared (first use in this function)
  275 |         LV_LOG_ERROR("write failed: %d", errno);
      |                                          ^~~~~
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_log.h:123:98: note: in definition of macro ‘LV_LOG_ERROR’
  123 | #    define LV_LOG_ERROR(...) lv_log_add(LV_LOG_LEVEL_ERROR, LV_LOG_FILE, LV_LOG_LINE, __func__, __VA_ARGS__)
      |                                                                                                  ^~~~~~~~~~~
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:32:1: note: ‘errno’ is defined in header ‘<errno.h>’; this is probably fixable by adding ‘#include <errno.h>’
   31 | #include "../../../misc/lv_area_private.h"
  +++ |+#include <errno.h>
   32 | 
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:275:42: note: each undeclared identifier is reported only once for each function it appears in
  275 |         LV_LOG_ERROR("write failed: %d", errno);
      |                                          ^~~~~
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_log.h:123:98: note: in definition of macro ‘LV_LOG_ERROR’
  123 | #    define LV_LOG_ERROR(...) lv_log_add(LV_LOG_LEVEL_ERROR, LV_LOG_FILE, LV_LOG_LINE, __func__, __VA_ARGS__)
      |                                                                                                  ^~~~~~~~~~~
make[2]: *** [lvgl/CMakeFiles/lvgl.dir/build.make:2402: lvgl/CMakeFiles/lvgl.dir/src/drivers/display/fb/lv_linux_fbdev.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_16.c.o
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_18.c.o
[ 45%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_22.c.o
[ 45%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_20.c.o
make[1]: *** [CMakeFiles/Makefile2:219: lvgl/CMakeFiles/lvgl.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
make: Leaving directory '/home/jkl/codes/lv_port_linux/build”

```

### How to reproduce?

### My Build step
```BASH
git clone https://github.com/lvgl/lv_port_linux.git
cd lv_port_linux/
git submodule update --init --recursive
mv configs/fbdev.defaults lv_conf.defaults 
echo "LV_LINUX_FBDEV_MMAP          0" >> lv_conf.defaults 
cmake -B build -S .
 make -C build -j20 
```
### The Error Outputs
```BASH
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_14.c.o
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_14_aligned.c.o
In file included from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_assert.h:17,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_color.h:17,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/lv_display.h:19,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.h:17,
                 from /home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:9:
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c: In function ‘write_to_fb’:
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:275:42: error: ‘errno’ undeclared (first use in this function)
  275 |         LV_LOG_ERROR("write failed: %d", errno);
      |                                          ^~~~~
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_log.h:123:98: note: in definition of macro ‘LV_LOG_ERROR’
  123 | #    define LV_LOG_ERROR(...) lv_log_add(LV_LOG_LEVEL_ERROR, LV_LOG_FILE, LV_LOG_LINE, __func__, __VA_ARGS__)
      |                                                                                                  ^~~~~~~~~~~
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:32:1: note: ‘errno’ is defined in header ‘<errno.h>’; this is probably fixable by adding ‘#include <errno.h>’
   31 | #include "../../../misc/lv_area_private.h"
  +++ |+#include <errno.h>
   32 | 
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/lv_linux_fbdev.c:275:42: note: each undeclared identifier is reported only once for each function it appears in
  275 |         LV_LOG_ERROR("write failed: %d", errno);
      |                                          ^~~~~
/home/jkl/codes/lv_port_linux/lvgl/src/drivers/display/fb/../../../display/../misc/lv_log.h:123:98: note: in definition of macro ‘LV_LOG_ERROR’
  123 | #    define LV_LOG_ERROR(...) lv_log_add(LV_LOG_LEVEL_ERROR, LV_LOG_FILE, LV_LOG_LINE, __func__, __VA_ARGS__)
      |                                                                                                  ^~~~~~~~~~~
make[2]: *** [lvgl/CMakeFiles/lvgl.dir/build.make:2402: lvgl/CMakeFiles/lvgl.dir/src/drivers/display/fb/lv_linux_fbdev.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_16.c.o
[ 44%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_18.c.o
[ 45%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_22.c.o
[ 45%] Building C object lvgl/CMakeFiles/lvgl.dir/src/font/lv_font_montserrat_20.c.o
make[1]: *** [CMakeFiles/Makefile2:219: lvgl/CMakeFiles/lvgl.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
make: Leaving directory '/home/jkl/codes/lv_port_linux/build'

```
### My Fix
```DIFF
diff --git a/src/drivers/display/fb/lv_linux_fbdev.c b/src/drivers/display/fb/lv_linux_fbdev.c
index 41c05037c..0ee9df627 100644
--- a/src/drivers/display/fb/lv_linux_fbdev.c
+++ b/src/drivers/display/fb/lv_linux_fbdev.c
@@ -17,6 +17,7 @@
 #include <sys/mman.h>
 #include <sys/ioctl.h>
 #include <time.h>
+#include <errno.h>
 
 #if LV_LINUX_FBDEV_BSD
     #include <sys/fcntl.h>
```

--- a/src/drivers/display/fb/lv_linux_fbdev.c
+++ b/src/drivers/display/fb/lv_linux_fbdev.c
@@ -17,6 +17,7 @@
 #include <sys/mman.h>
 #include <sys/ioctl.h>
 #include <time.h>
+#include <errno.h>
 
 #if LV_LINUX_FBDEV_BSD
     #include <sys/fcntl.h>

